lib/ios/RNNBottomTabsPresenter.m

```diff
@@ -9,6 +9,9 @@ - (void)applyOptionsOnInit:(RNNNavigationOptions *)options {
     UITabBarController *bottomTabs = self.boundViewController;
     RNNNavigationOptions *withDefault = [options withDefault:[self defaultOptions]];
     [bottomTabs setCurrentTabIndex:[withDefault.bottomTabs.currentTabIndex getWithDefaultValue:0]];
+	if ([[withDefault.bottomTabs.titleDisplayMode getWithDefaultValue:@"alwaysShow"] isEqualToString:@"alwaysHide"]) {
+		[bottomTabs setTabItemImagesCentered];
```

Comment:
Sorry for nitpicking - but this public method isn't a setter. It should probably be named `centerTabItems` as it does a job - centre both icons and badge (not only images)